### PR TITLE
chore: update sublime-package.json

### PR DESCRIPTION
**Pull request by husanjun**

### CODE REVIEW

**Comment by jfcherng** [comment]
Thanks!

File: sublime-package.json [hunk 1]
```diff
@@ -46,12 +46,17 @@
                     "intelephense.compatibility.correctForBaseClassStaticUnionTypes": {
                       "type": "boolean",
                       "default": true,
-                      "markdownDescription": "Resolves `BaseClass|static` union types to `static` instead of `BaseClass`."
+                      "description": "Resolves `BaseClass|static` union types to `static` instead of `BaseClass`.",
```
Comment (rchl) [inline sublime-package.json:49]: Why changing from `markdownDescription` to `description` and loosing markdown formatting?

Comment (husanjun) [reply sublime-package.json:49]: Copy of intelephense, I will fix it

Comment (jfcherng) [reply sublime-package.json:49]: > Copy of intelephense, I will fix it

may file a pr to intelephense too

**Pull request merged**